### ISSUE TITLE
Snowflake SEMANTIC_VIEW uses incorrect LT01 bracket spacing

### ISSUE THREAD

**Issue by jaysobel** [sqlfluff/sqlfluff#8458]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

The dedicated Snowflake `SEMANTIC_VIEW(...)` grammar added in #8447 is correctly parsed as a `semantic_view` rather than a generic function. Because the new segment has no corresponding layout policy, LT01 applies the default keyword-to-bracket spacing and reports Snowflake's documented syntax:

```sql
SELECT *
FROM SEMANTIC_VIEW(
    sales_analysis
    DIMENSIONS orders.order_date
    METRICS orders.revenue
);
```

Snowflake's syntax reference and examples consistently use the attached form: https://docs.snowflake.com/en/sql-reference/constructs/semantic_view

### Expected Behaviour

LT01 should treat the inline boundary between `SEMANTIC_VIEW` and its opening bracket as touching, matching Snowflake's documented syntax and other attached-bracket constructs. `sqlfluff fix` should remove an accidental inline space.

### Observed Behaviour

LT01 reports `Expected single whitespace between 'SEMANTIC_VIEW' keyword and start bracket '('` for `SEMANTIC_VIEW(`. Running `sqlfluff fix` changes it to `SEMANTIC_VIEW (`. Conversely, LT01 accepts the spaced form.

### How to reproduce

On current `main` after #8447, save the SQL above as `semantic_view.sql`, then run:

```console
sqlfluff lint semantic_view.sql --dialect snowflake --rules LT01
sqlfluff fix semantic_view.sql --dialect snowflake --rules LT01
```

### Dialect

Snowflake

### Version

Current `main` at `d3e8358bc` (after #8447); Python 3.10.13.

### Configuration

No project configuration. The commands specify `--dialect snowflake --rules LT01`.

### Are you willing to work on and submit a PR to address the issue?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
